src/testing/BoardVisibilityOption.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { cleanup, fireEvent, render } from "@testing-library/react";
+import { BoardVisibilityOption } from "../components/boardPage/BoardHeader/BoardVisibilityOption";
+
+afterEach(cleanup);
+
+let boardVisibility = true;
+jest.mock("../hooks/useBoardState", () => {
+  const useBoardState = () => {
+    const changeBoardVisibility = (newState: boolean) => {
+      board.isPublic = newState;
+      boardVisibility = board.isPublic;
+    };
+
+    const board = { isPublic: boardVisibility };
+
+    return { changeBoardVisibility, board };
+  };
+  return { useBoardState };
```

Comment:
Not asking to do this. But please just consider this for following changes and for the PT: 

Whenever changing the value of a boolean through a function, toggling through returning its opposite with the ! flag is recommended 

```
const changeBoardVisibility = () => setState( current => !current );
```

Reply:
Got It 👍